### DISCUSSION TITLE
Authentication

### DISCUSSION THREAD

**Discussion by lajixuebiancheng** [community/community#52025] [GitHub Education]
My credentials were denied more than 20 times, but I had followed them to the letter. 
it just say, Unfortunately, we were unable to verify your academic affiliation based on the information you submitted

**Comment by samus-aran** [comment]
Hi @lajixuebiancheng 👋 

Ensure you select the most appropriate **evidence type** in the dropdown. The order of options indicates the preference of evidence with Academic Email being at the top. If you don't have an Academic email or Student ID, do try and acquire evidence that is positioned higher rather than lower in that option list.

If you have been repetitively unsuccessful using a particular evidence type (e.g. Student ID) we would recommend that you acquire and submit an alternative piece of evidence that can verify your student status.

If your original document is not in English then we advise you to take a photo of the original next to an English translation. This also include your profile information.
Ensure your evidence correspond to details in your GitHub profile. The verification process looks at your GitHub username, email and name ( as well as other points) to cross reference the evidence submitted. Do you use a variation of your name or a nickname?  It’s these small but important points to consider when applying that can make a much smoother process.

Let us know how you get on 👍
